docs/samples/Microsoft.ML.Samples/Static/FastTreeRegression.cs

```diff
@@ -1,20 +1,15 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-        // the alignment of the usings with the methods is intentional so they can display on the same level in the docs site. 
-        using Microsoft.ML.Runtime.Data;
-        using Microsoft.ML.Trainers.FastTree;
-        using Microsoft.ML.StaticPipe;
-        using System;
-        using System.Linq;
+﻿using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Trainers.FastTree;
+using Microsoft.ML.StaticPipe;
+using System;
+using System.Linq;
 
 // NOTE: WHEN ADDING TO THE FILE, ALWAYS APPEND TO THE END OF IT. 
 // If you change the existinc content, check that the files referencing it in the XML documentation are still correct, as they reference
```

Comment:
```suggestion
// If you change the existing content, check that the files referencing it in the XML documentation are still correct, as they reference
```

Reply:
Above we already discussed these comments as obsoleted - would it be better to completely remove them from samples? I may do that in this same pull request.